### ISSUE TITLE
removed in django 1.9 warning

### ISSUE THREAD

**Issue by hyusetiawan** [nitely/django-djconfig#15]
I am using Sprit, everything works, i have successfully integrated it with  my app however, when I do runserver, I keep getting the following:

/Projects/env3/lib/python3.4/site-packages/djconfig/models.py:10: RemovedInDjango19Warning: Model class djconfig.models.Config doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.
  class Config(models.Model):

this is my installed apps:

```
from spirit.settings import *

INSTALLED_APPS = INSTALLED_APPS + [
    'rest_framework',
    'django.contrib.sites',
    'myapp',
    'reversion',
    'django_rq',
    'allauth',
    'allauth.account',
    'allauth.socialaccount',
    'allauth.socialaccount.providers.facebook',
    'allauth.socialaccount.providers.linkedin',
    'allauth.socialaccount.providers.google',
    'allauth.socialaccount.providers.twitter',
    'allauth.socialaccount.providers.github'
]
```

**Comment by nitely** [comment]
It's a harmless warning. I've fixed it a while ago, but I haven't make a new release.

**Comment by nitely** [comment]
This should be fixed on v0.5.0
